tests/frontDesk/visits.js

```diff
@@ -163,4 +163,9 @@ step("Validate obs <form> on the patient clinical dashboard", async function (fo
     await taikoHelper.repeatUntilNotFound($("#overlay"))
     await taikoHelper.validateFormFromFile(obsFormValues.ObservationFormDetails, obsFormValues.ObservationFormName)
     await click($('.ngdialog-close'))
-});
\ No newline at end of file
+});
+
+step("Verify diagnosis in patient dashboard", async function() {
```

Comment:
step("Verify diagnosis in patient clinical dashboard", async function () {
    var medicalDiagnosis = gauge.dataStore.scenarioStore.get("medicalDiagnosis")
    assert.ok(await text(medicalDiagnosis.diagnosis.diagnosisName, toLeftOf(medicalDiagnosis.diagnosis.certainty, toRightOf(medicalDiagnosis.diagnosis.order)), within($("#Diagnoses"))).exists())
});

Cannot use this function because here we are getting the value from the json file,its different

Reply:
once you identified the Diagnosis which is not present in Openmrs.
put that in data store gauge.dataStore.scenarioStore.put("medicalDiagnosis",jsonObject);
and you can reuse this method.

Reply:
okayy but what about this line "assert.ok(await text(medicalDiagnosis.diagnosis.diagnosisName, toLeftOf(medicalDiagnosis.diagnosis.certainty, toRightOf(medicalDiagnosis.diagnosis.order)), within($("#Diagnoses"))).exists())
});"
medicalDiagnosis.diagnosis.diagnosisName->how to reuse this line,it is checking in the json file which have diagnosis and diagnosis name

Reply:
Create a JSON object like that.

Reply:
done